src/Extensions/ElementalAreaUsedOnTableExtension.php

```diff
@@ -36,6 +36,9 @@ public function updateUsageExcludedClasses(array &$excludedClasses)
      */
     public function updateUsage(ArrayList &$usage, DataObject &$record)
     {
+        Deprecation::withNoReplacement(function () {
```

Comment:
This method call requires us to bump the framework dependency

Reply:
Updated